app/services/request.ts

```diff
@@ -83,3 +83,12 @@ export const deleteRequest = async (id?: number): Promise<[Request[], null] | [n
     return handleAxiosError(err);
   }
 };
+
+export const updateRequestMetadata = async (data: Request): Promise<[Request, null] | [null, AxiosError]> => {
+  try {
+    const result = await instance.put('request-metadata', data).then((res) => res.data);
```

Comment:
Naive question, is there a reason we don't have a get for the metadata? Or is the metadata a purely used for backend/metabase/database purposes?

Reply:
oh, the API endpoint is to update integration metadata that requires a request body.
and `POST` is mostly for creating entity and `PUT` is to update, so used `PUT` in this case.